kyuubi-main/src/test/scala/org/apache/kyuubi/server/MiniYarnService.scala

```diff
@@ -97,11 +100,14 @@ class MiniYarnService(name: String) extends AbstractService(name) {
     super.stop()
```

Comment:
Shall we delete the temporary file after stop ?

Reply:
thanks, added.

The file created by Utils.createTempDir should be deleted on exit.